packages/devtools_app/lib/src/network/flutter/http_cookies_tab.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../http.dart';
+import '../http_request_data.dart';
+
+/// A [Widget] which displays [Cookie] information in a tab.
+class HttpRequestCookiesTab extends StatelessWidget {
+  const HttpRequestCookiesTab(this.data);
+
+  DataColumn _buildColumn(String title, {bool numeric = false}) => DataColumn(
+      label: Expanded(
+        child: Text(
+          title,
+          style: _headerTextStyle,
+          overflow: TextOverflow.fade,
+        ),
+      ),
+      numeric: numeric);
+
+  DataRow _buildRow(int index, Cookie cookie, {bool requestCookies = false}) =>
+      DataRow.byIndex(index: index, cells: <DataCell>[
+        _buildCell(cookie.name),
+        _buildCell(cookie.value),
+        if (!requestCookies) ...[
+          _buildCell(cookie.domain ?? 'N/A'),
+          _buildCell(cookie.path ?? 'N/A'),
+          _buildCell(cookie.expires?.toString() ?? 'N/A'),
+          _buildCell(cookie.value.length.toString()),
+          _buildIconCell(!cookie.httpOnly ? Icons.check : Icons.close),
+          _buildIconCell(!cookie.secure ? Icons.check : Icons.close),
+        ]
+      ]);
+
+  DataCell _buildCell(String value) => DataCell(Text(value));
+  DataCell _buildIconCell(IconData icon) => DataCell(Icon(icon));
+
+  Widget _buildCookiesTable(
+      String title, List<Cookie> cookies, BoxConstraints constraints,
+      {bool requestCookies = false}) {
+    return Column(
+      children: <Widget>[
+        Align(
+          alignment: Alignment.centerLeft,
+          child: Text(
+            title,
+            style: const TextStyle(
+                color: Colors.lightBlue,
+                fontSize: 18,
+                fontWeight: FontWeight.bold),
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(bottom: 24.0),
```

Comment:
nit: make 24.0 a constant with a name that documents the intent of this quite large amount of padding.